sf-docker/sf-observability/docker-compose.yml

```diff
@@ -0,0 +1,214 @@
+version: "3.6"
+
+services:
+  obsminio:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    image: "minio/minio:RELEASE.2022-06-02T02-11-04Z"
+    profiles: [singlenode, multinode-aggregator]
+    command: server /data --console-address ":9001"
+    expose:
+      - 9000
+    ports:
+      - "9901:9001"
+    volumes:
+      - minio_data:/data
+    networks:
+      observability:
+    restart: unless-stopped
+
+  obsminio_createbuckets:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    image: minio/mc
+    profiles: [singlenode, multinode-aggregator]
+    depends_on:
+      obsminio:
+        condition: service_started
+    entrypoint: >
+      /bin/sh -c "
+      sleep 10;
+      /usr/bin/mc config host add obsminio http://obsminio:9000 minioadmin minioadmin;
+      /usr/bin/mc mb obsminio/loki;
+      /usr/bin/mc policy set public obsminio/loki;
+      exit 0;
+      "
+    networks:
+      observability:
+  
+  grafana:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    profiles: [singlenode, multinode-aggregator]
+    image: grafana/grafana:10.1.4
+    ports:
+      - 9900:3000
+    volumes:
+      - ./configuration/grafana/datasources:/etc/grafana/provisioning/datasources
+      - ./configuration/grafana/dashboards:/etc/grafana/provisioning/dashboards
```

Comment:
Please include these files using docker compose `configs`, also make sure to specify file attributes.

Reply:
I understand why it needs to be done, while solving other configs. But in this case
I need to do config for every file: Which are 7 files right now.

and then in every service I need to use long syntax
https://docs.docker.com/reference/compose-file/services/#long-syntax
to mount it in correct folder

are we sure we want do it this way ?:)